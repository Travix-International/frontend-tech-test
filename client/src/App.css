/* .App {
  text-align: center;
}

.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 40vmin;
  pointer-events: none;
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
} */

#root {
  background-color:black;
}

.appContainer {
  display: flex;
  flex-direction: column;
  height: 100vh;
  overflow-y: hidden;
}

.alertContainer {
  display: flex;
  flex-direction: row;
  justify-content: center;
  align-items: center;
}

.alertContainer .alert {
  display: flex;
  flex-direction: row;
  justify-content: center;
}

.addTaskContainer{
  display: flex;
  flex-direction: row;
  justify-content: center;
  align-items: center;
  height: 5vh
}

.sortFilterContainer {
  display: flex;
  flex-direction: row;
  justify-content: center;
  align-items: center;
  height: 5vh
}

.taskListContainer {
  display: flex;
  flex-direction: row;
  justify-content: center;
  height: 70vh
}