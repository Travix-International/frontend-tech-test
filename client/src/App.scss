.title {
    font-size: 3rem;
    color: red;
}