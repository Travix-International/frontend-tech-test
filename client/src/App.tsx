import React from 'react';
import styles from './App.scss';

class App extends React.Component<{}> {
    render() {
        return (
            <div className="App">
                <h1 className={styles.title}>Hello World!!</h1>
            </div>
        );
    }
}

export default App;