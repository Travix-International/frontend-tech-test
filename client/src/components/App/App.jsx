import React from 'react';
import { Switch, Route, Link } from 'react-router-dom';

import TaskDetails from '../../containers/TaskDetails/TaskDetails';
import TaskManager from '../../containers/TaskManager/TaskManager';
import Container from '../UI/Container/Container';
import styles from './App.module.scss';
import routes from '../../constants/routes';


const App = () => (
  <div className={styles.App}>
    <Container>
      <h1>
        <Link className={styles.Logo} to="/">Task Manager</Link>
      </h1>
      <Switch>
        <Route component={TaskDetails} exact path={routes.taskPath} />
        <Route component={TaskManager} exact path={routes.tasks} />
      </Switch>
    </Container>
    
  </div>
);

export default App;
