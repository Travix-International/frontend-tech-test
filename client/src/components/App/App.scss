@import url('https://fonts.googleapis.com/css?family=Sawarabi+Mincho');
@import '../../scss/vars.scss';
body {
  background-color: $yellow;
  font-size: 16px;
  font-family: 'Sawarabi Mincho', sans-serif;
}