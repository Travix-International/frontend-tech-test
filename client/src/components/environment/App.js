import React, { Component } from 'react'
import { Provider } from 'react-redux'

import Tasks from '../ecosystems/Tasks/index'
import StoreConfigurator from '../../ducks/StoreConfigurator'

const storeConfigurator = new StoreConfigurator()
const store = storeConfigurator.configure()

export default class App extends Component {
  render() {
    return (
      <Provider store={store}>
        <Tasks />
      </Provider>
    )
  }
}
