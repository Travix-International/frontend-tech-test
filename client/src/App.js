import React, { Component } from 'react';
import './App.scss';
import { Provider } from 'react-redux';
import store from './store';
import TodoForm from './components/TodoForm';
import List from './components/List';

export class App extends Component {
  render() {
    return (
      <Provider store={store}>
        <div className="App">
          <div className="container">
            <div className="row">
              <div className="col-md-4">
                <TodoForm />
              </div>
              <div className="col-md-8">
                <List />
              </div>
            </div>
          </div>
        </div>
      </Provider>
    );
  }
}

export default App;
