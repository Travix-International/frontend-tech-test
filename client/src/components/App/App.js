import React from 'react'
import LayoutContainer from '../../containers/LayoutContainer/LayoutContainer'

import './App.scss'

const App = () => <LayoutContainer />

export default App
