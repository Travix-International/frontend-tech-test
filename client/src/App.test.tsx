import React from 'react';
import App from './App';

describe('test', () => {
    it('should pass', () => {
        expect(<App />).toMatchSnapshot();
    })
});