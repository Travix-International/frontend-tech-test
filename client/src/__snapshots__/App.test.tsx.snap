// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test should pass 1`] = `<App />`;
